### PR TITLE
Add ChatGPT/Azure OpenAI sample

### PR DESCRIPTION
**Pull request by Danieladu**
Add OpenAI and Azure OpenAI Sample project (using Chat GPT API)

### CODE REVIEW

File: samples/csharp_dotnetcore/83.openai/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "MicrosoftAppType": "",
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
+  "MicrosoftAppTenantId": "",
+  "OpenAIKey": "",
```
Comment (boydc2014) [inline samples/csharp_dotnetcore/83.openai/appsettings.json:6]: If I fill in both open ai key and azure ai key, which one will be used?

Comment (Danieladu) [reply samples/csharp_dotnetcore/83.openai/appsettings.json:6]: Both are used

**Comment by Danieladu** [comment]
@microsoft/botframework-sdk may you take a look? Thanks

**Comment by tracyboehrer** [comment]
@Danieladu I hate how you spent time on this, but we made a decision to not add anything OpenAI related to SDK.  It opens a flood gate that we aren't staffed to support and grow.  This could change so I'd keep this branch around.

**Pull request closed without merging**